CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+### citus v13.0.0 (January 17, 2025) ###
+
+* Adds support for PostgreSQL 17 (#7699, #7661)
+
+* Adds JSON_TABLE() support in distributed queries (#7816)
+
+* Propagates MERGE ... WHEN NOT MATCHED BY SOURCE (#7807)
```

Comment:
```suggestion
* Propagates `MERGE .. WHEN NOT MATCHED BY SOURCE` (#7807)
```

Reply:
Good suggestions Hanefi, I forgot we're in .md :)